ui/app/src/components/PreviewSearchPanel/PreviewSearchPanel.tsx

```diff
@@ -137,13 +145,17 @@ export function PreviewSearchPanel() {
 
   const onSearch = useCallback(
     (keywords: string = '', options?: ComponentsContentTypeParams) => {
+      let sub;
```

Comment:
Missing type; but moreover, I don't see a point in declaring this here above instead of doing it inline with the call that would automatically give it its type.